src/vehicle.h

```diff
@@ -889,6 +890,7 @@ class vehicle : public JsonSerializer, public JsonDeserializer
     int aisle_lights_epower = 0;
     int recharger_epower    = 0; // total power consumed by rechargers
     int camera_epower       = 0; // power consumed by camera system
+    int plow_engine_drag       = 0; //The amount of friction applied by plows on the vehicle.
```

Comment:
Perhaps make this a generic "accessory_drag" value, for example when we add trailers the trailer would add drag as well.